src/SDKs/Batch/DataPlane/Tools/ConfigureAwaitAnalyzer/ConfigureAwaitAnalyzer.Tests/ConfigureAwaitAnalyzer.Tests.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!--<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />-->
+  <PropertyGroup>
```

Comment:
I copied almost all of this from `test.props` -- so if I don't need it no other test project needs it too.

I didn't want to use `test.props` because it includes `DisableTestRunParallel.cs` which since these tests are all in-memory unit tests is pointless.

If you want, you can update `test.props` to have a flavor that doesn't have `DisableTestRunParallel.cs` and I can use that. Although I am not sure why we are bothering to sign our tests either.